### ISSUE TITLE
Private Access Tokens will impact whether or not Captchas will be needed to be done

### ISSUE THREAD

**Issue by ollie-iterators** [w3c/wcag#2524]
Private Access Tokens will allow users to bypass Captchas, which would mean that they would not need to be used (and won't need to be accessible). This should be noted in the techniques in wcag that are about CAPTCHA's.

See this Link: https://www.ietf.org/archive/id/draft-private-access-tokens-01.html

**Comment by a11ydoer** [comment]
Hi @ollie-iterators Thanks for the URL information about Private Access Tokens. I am trying to find a way to incorporate priviate access token into SC 3.3.7 if that is relevant. 

When I see the information you shared, I see three methods for authentification [1.1.1](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#section-1.1.1).  [Rate-limited Access](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#name-rate-limited-access)

[1.1.2](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#section-1.1.2).  [Client Geo-Location](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#name-client-geo-location)

[1.1.3](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#section-1.1.3).  [Private Client Authentication](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#name-private-client-authenticati)

One potential cross over with SC 3.3.7 may be the token challenge but I don't think this is directly related end users https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#name-token-challenge

Would you kindly explain what is the interaction point between SC 3.3.7 and Private Access Token methods?

**Comment by a11ydoer** [comment]
Here is the info I found regarding [user involvement](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html#name-user-interaction) from the linked doc. 

_"Private Access Tokens are designed to be used without explicit user involvement. Since tokens are only valid for a single origin and in response to a specific challenge, there is no need for a user to manage a limited pool of tokens across origins."_

**Comment by a11ydoer** [comment]
@mbgower @alastc suggested to add "There are a number of technologies that can be employed to to prevent scripted abuse of the  authentication process. [LIST choices including private access tokens here]. However, because most of these systems can result in... they may still result in a CAPTCHA being displayed to users. So none of them are sufficient techniques to meet this SC, although they may lessen the impact…" I will work on PR.

**Comment by a11ydoer** [comment]
@alastc Can you review the [RR 2983](https://github.com/w3c/wcag/pull/2983) and merge?
